Begin restructuring installation docs (#64429)
gravitational/teleport: docs/pages/installation/single-machine/single-machine.mdx

---
title: Installing Teleport on a Single Machine
sidebar_label: Single Machine
description: How to install Teleport and Teleport's client tools on your platform, including binaries and instructions for Docker and Helm.
videoBanner: fjk29wqXm1A
tags:
- reference
- platform-wide
---

The guides in this section explain how to install Teleport on a single machine.

Teleport installations include the following binaries:

|Binary|Purpose|
|---|---|
|`teleport`|Main Teleport binary that runs the Auth Service, Proxy Service, and Teleport Agent services.|
|`tctl`|Admin tool for managing Teleport resources.|
|`tsh`|Client tool for accessing Teleport-protected infrastructure.|
|`tbot`|CLI for running Teleport Machine & Workload Identity Bots.|

> |`teleport-update`|Tool for operating Teleport Managed Updates.|

Reviewer: Is `fdpass-teleport` there too?

Author: My thought was to leave this out because it's for internal use, but I can also see how it would make sense to describe every binary the user receives in an installation. I'll add a line.

> Whether you are installing Teleport as a cluster (the Auth Service and Proxy

Reviewer: Nit: auth, proxy, and agents are all part of a Teleport cluster.

```suggestion
Whether you are installing the Teleport control plane (the Auth Service and Proxy
```
